kivy/core/camera/camera_opencv.py

```diff
@@ -97,12 +146,9 @@ def _update(self, dt):
 
     def start(self):
         super(CameraOpenCV, self).start()
-        if self._update_ev is not None:
-            self._update_ev.cancel()
-        self._update_ev = Clock.schedule_interval(self._update, self.fps)
+        Clock.unschedule(self._update)
```

Comment:
Referencing the event instead of the method is much faster when unscheduling.
If the reason of the change was only to refactor, it should not be done in this case.